front/app/containers/Admin/pagesAndMenu/containers/ContentBuilder/components/Widgets/Selection/index.tsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+
+import { Multiloc } from 'typings';
+
+import useAdminPublicationsByIds from 'api/admin_publications/useAdminPublicationsByIds';
+
+import useLocalize from 'hooks/useLocalize';
+
+import AdminPubsCarrousel from '../_shared/AdminPubsCarrousel';
+import EmptyState from '../_shared/EmptyState';
+
+import messages from './messages';
+import Settings from './Settings';
+
+interface Props {
+  titleMultiloc: Multiloc;
+  adminPublicationIds: string[];
+}
+
+const Selection = ({ titleMultiloc, adminPublicationIds }: Props) => {
+  const localize = useLocalize();
+  const { data, hasNextPage, fetchNextPage } = useAdminPublicationsByIds(
+    {
+      ids: adminPublicationIds,
+      pageSize: 6,
+    },
+    {
+      enabled: adminPublicationIds.length > 0,
+    }
+  );
+
+  const adminPublications = data?.pages.map((page) => page.data).flat();
+
+  const showEmptyState =
+    adminPublicationIds.length === 0 || adminPublications?.length === 0;
+
+  if (showEmptyState) {
+    return (
+      <EmptyState titleMultiloc={titleMultiloc} explanation={messages.noData} />
+    );
+  }
+
+  if (!adminPublications) return null;
+
+  return (
+    <AdminPubsCarrousel
```

Comment:
How do you feel about renaming this to AdminPublicationsCarrousel? Personally I don't think it's too long, and I had to think for a second what AdminPubs were :joy:

Reply:
Done!